component/class/defaults.yml

```diff
@@ -54,7 +62,7 @@ parameters:
     appuioManaged: true
     tenantID: ${cluster:tenant}
     quotasEnabled: false
-    grpcEndpoint: unix-abstract:crossplane/fn/default.sock
+    grpcEndpoint: host.docker.internal:9443
```

Comment:
This is probably wrong

Reply:
This is now only used for the proxy in debugging, the endpoint is completely crossplane managed.